diff --git a/.DS_Store b/.DS_Store
index f429048..2bbc088 100644
Binary files a/.DS_Store and b/.DS_Store differ
diff --git a/.gitignore b/.gitignore
index e69de29..4e0603f 100644
--- a/.gitignore
+++ b/.gitignore
@@ -0,0 +1,2 @@
+__pycache__
+runs/
\ No newline at end of file
diff --git a/utils.py b/utils.py
index 986d0a8..3754c2a 100644
--- a/utils.py
+++ b/utils.py
@@ -8,7 +8,7 @@ class NPK_Args:
     """Environment seed"""
     seed: int = 0
     """Path"""
-    path: Path = Path("/Users/wsolow/Projects/agaid_crop_simulator/")
+    path: str = "/Users/wsolow/Projects/agaid_crop_simulator/"
 
     """Output Variables"""
     output_vars: list = field(default_factory = lambda: ['DVS', 'LAI', 'RD', 'WSO', 'NuptakeTotal', 'PuptakeTotal', 'KuptakeTotal', 'NAVAIL', 'PAVAIL', 'KAVAIL', 'WC'])
diff --git a/wandb/debug-internal.log b/wandb/debug-internal.log
index 2d235b3..b83e7c5 120000
--- a/wandb/debug-internal.log
+++ b/wandb/debug-internal.log
@@ -1 +1 @@
-run-20240828_131037-jkf6txha/logs/debug-internal.log
\ No newline at end of file
+run-20240828_141611-7paxyg76/logs/debug-internal.log
\ No newline at end of file
diff --git a/wandb/debug.log b/wandb/debug.log
index 95008ad..e5ead56 120000
--- a/wandb/debug.log
+++ b/wandb/debug.log
@@ -1 +1 @@
-run-20240828_131037-jkf6txha/logs/debug.log
\ No newline at end of file
+run-20240828_141611-7paxyg76/logs/debug.log
\ No newline at end of file
diff --git a/wandb/latest-run b/wandb/latest-run
index 849965e..3a53fb1 120000
--- a/wandb/latest-run
+++ b/wandb/latest-run
@@ -1 +1 @@
-run-20240828_131037-jkf6txha
\ No newline at end of file
+run-20240828_141611-7paxyg76
\ No newline at end of file
